### ISSUE TITLE
Ctrl+Enter doesn't quite release

### ISSUE THREAD

**Issue by phet-steele** [phetsims/balloons-and-static-electricity#138]
Using Ctrl+Enter to release a balloon does not work quite right. The balloon does get dropped, but the selection box around it stays black. Movement controls do not affect the balloon at this point.

For phetsims/tasks/issues/566

**Comment by terracoda** [comment]
I think we can drop the Ctrl+Enter interaction pattern. I included this pattern because it is the default interaction for submitting a form, but only one participant in the tried it out, though not all participants found the instructions to for that interaction. 
We might just want to stick with Spacebar. It will simplify writing the help documentation as well.

**Comment by jessegreenberg** [comment]
> We might just want to stick with Spacebar. It will simplify writing the help documentation as well.

Ok, sounds good.  While fixing this issue, I will change the interaction so that only the spacebar drops the balloon.

**Comment by jessegreenberg** [comment]
Fixed in commit above, 'ctrl + enter' has been from possible 'release' interactions.  Spacebar is the only way to release the balloon now, and it was working well.  Closing.
